Signal/src/contact/OWSContactsManager.m

```diff
@@ -46,6 +49,41 @@ - (id)init {
     return self;
 }
 
+- (NSDictionary *)latestContactsById
+{
+    @synchronized(self)
+    {
+        return _latestContactsById;
```

Comment:
nit: seems like we should initialize `_latestContactsById` to an empty dict in `init`